sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/storage/DatabaseManager.java

```diff
@@ -425,7 +433,7 @@ long getMaxSize() {
          * @param newVersion new version of the schema.
          * @return true if upgrade was managed, false to drop/create table.
          */
-        @SuppressWarnings({"BooleanMethodIsAlwaysInverted", "unused"})
+        @SuppressWarnings("BooleanMethodIsAlwaysInverted")
```

Comment:
Is `BooleanMethodIsAlwaysInverted` still needed?

Reply:
Yes. Look at the single use:
https://github.com/Microsoft/AppCenter-SDK-Android/blob/42dc17f2b4670849d6e4f9e74297c163046d6e26/sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/storage/DatabaseManager.java#L141